### DISCUSSION TITLE
Captcha for the Visually Impaired

### DISCUSSION THREAD

**Discussion by Windthin** [e107inc/e107#4641] [Q&A Plugins]
## Question
I see several Captcha options available; but I do not know which, if any, is best for somebody with visual impairment; one of my admins has issues there, and I am hoping there is an option with an audio alternative or something similar that she can use?

## Expected Source
I looked on here first to see if anybody mentioned this, and nobody had.

## Additional Context
We've just undergone a major upgrade from 1.04 to 2.x and are only for the first time adding captcha to the site, so I want to make sure I do it in a way that doesn't exclude one of my admins.

**Comment by Moc** [comment]
Hi @Windthin 

These type of questions are best asked on e107help.org for now :)
Closing it here.

**Reply by Moc** [reply]
@Windthin Recently we opened up this discussion area on Github. We're transitioning away from e107help.org. I've therefore moved your question to the new Discussion area :)

**Reply by Windthin** [reply]
Much appreciated!

**Comment by Moc** [comment]
You may want to check out the Visual Captcha plugin. It uses pictures as well as audio. 

It can be found through the `Admin Area > Plugin Manager > Find Plugins` section (search for  "Visual Captcha"). Or you can download it directly from the Github repository: https://github.com/e107inc/visualcaptcha

I am not sure if it's fully functional with the latest version though (because of https://github.com/e107inc/visualcaptcha/issues/3). There also appears to be discussion about it being resistant to bots. It appears bots can easily pass the verification. 

Alternatively, maybe the admin you are referring to may have suggestions for which captcha systems seem to work best for her? Using that information, we could see if an integration of that system with e107 can be made.
